src/MetricsUploader/CaptureMetricTest.cpp

```diff
@@ -40,8 +40,10 @@ const CaptureCompleteData kTestCompleteData{
     109 /*number_of_manual_tracked_value_timers*/,
     110 /*number_of_callstack_samples*/,
     111 /*number_of_unwinding_errors*/,
-    std::filesystem::path{"/test/path"} /*file_path*/
-};
+    std::filesystem::path{"/test/path"} /*file_path*/,
+    OrbitCaptureData_TargetProcessStateAfterCapture_TARGET_PROCESS_STATE_AFTER_CAPTURE_CRASHED /*target_process_state_after_capture*/
```

Comment:
Nit: prefer the :: scope resolution operator over underscore. 
So `OrbitCaptureData::TargetProcessStateAfterCapture::TARGET_PROCESS_STATE_AFTER_CAPTURE_CRASHED`
And the save for the below.

Reply:
I tried around a bit - the best I found was 
`OrbitCaptureData::TARGET_PROCESS_STATE_AFTER_CAPTURE_CRASHED`

changed here and below.